release/rancher/rancher.go

```diff
@@ -836,12 +977,15 @@ func registryAuth(authURL, service, image string) (string, error) {
 	return auth.Token, nil
 }
 
-func rancherPrimeArtifact(url string) ([]string, error) {
+func remoteTextFileToSlice(url string) ([]string, error) {
 	httpClient := ecmHTTP.NewClient(time.Second * 15)
 	res, err := httpClient.Get(url)
 	if err != nil {
 		return nil, err
 	}
+	if res.StatusCode != 200 {
```

Comment:
Let's move this check below the defer call.

Reply:
Updated